lib/recipe/widgets/recipe_probes_screen.dart

```diff
@@ -0,0 +1,242 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:gibsonify/recipe/recipe.dart';
+import 'package:gibsonify_api/gibsonify_api.dart';
+import 'package:flutter_slidable/flutter_slidable.dart';
+import 'package:gibsonify/navigation/navigation.dart';
+import 'package:dropdown_search/dropdown_search.dart';
+
+class RecipeProbeScreen extends StatelessWidget {
```

Comment:
The filename is `recipe_probes_screen` but the main class is `RecipeProbeScreen` so I think since its multiple probes the class name can be changed to have the plural Probe**s**?